src/overview/assets/index.html

```diff
@@ -94,9 +94,9 @@ <h6 class="text-capitalize mb-2">
               </div>
               <div class="tab-pane fade" id="otherJdkPanel" role="tabpanel" aria-labelledby="otherJdkTab">
                 <ul id="jdkProviderList" class="list-unstyled">
-                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.azul.com%2Fdownloads%2Fazure-only%2Fzulu%2F%22" title="Recommended for Microsoft Azure Cloud and Azure Stack applications">Azul Zulu Enterprise for Azure</a></li>
-                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.oracle.com%2Ftechnetwork%2Fjava%2Fjavase%2Fdownloads%2Findex.html%22" title="Java SE Downloads by Oracle">Java SE Downloads by Oracle</a></li>
-                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fopenjdk.java.net%2Finstall%2F%22" title="OpenJDK">OpenJDK</a></li>
+                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.azul.com%2Fdownloads%2Fazure-only%2Fzulu%2F%22" title="Recommended for Microsoft Azure Cloud and Azure Stack applications">Azul Zulu Enterprise OpenJDK for Microsoft Azure</a></li>
+                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fdevelopers.redhat.com%2Fproducts%2Fopenjdk%2Fdownload%22" title="OpenJDK by Red Hat">Red Hat OpenJDK</a></li>
+                  <li><a href="command:java.helper.openUrl?%22https%3A%2F%2Fwww.oracle.com%2Ftechnetwork%2Fjava%2Fjavase%2Fdownloads%2Findex.html%22" title="OpenJDK by Oracle">Oracle OpenJDK</a></li>
```

Comment:
From the [link](https://www.oracle.com/technetwork/java/javase/downloads/index.html), I don't think it's the  "Oracle OpenJDK". 

> Oracle Customers and ISVs targeting Oracle LTS releases: **Oracle JDK** is Oracle's supported Java SE version for customers and for developing, testing, prototyping or demonstrating your Java applications.

> End users and developers looking for free JDK versions:[ Oracle OpenJDK ](https://jdk.java.net/11/)offers the same features and performance as Oracle JDK under the GPL license.